src/misc/TranslationKey.ts

```diff
@@ -1659,7 +1659,6 @@ export type TranslationKeyType =
 	| "whitelabelDomain_label"
 	| "whitelabelRegistrationCode_label"
 	| "whitelabelRegistrationEmailDomain_label"
-	| "whitelabelThemeDetected_msg"
```

Comment:
the key will come back on the next translation update, we should mark it as deprecated in Phrase at least